tests/test_name_version.py

```diff
@@ -1,7 +1,16 @@
-def test_get_app_and_version(cmd, hid):
-    if hid:
-        # for now it doesn't work with Speculos
-        app_name, version = cmd.get_app_and_version()
+from boilerplate_client.boilerplate_cmd import BoilerplateCommand
 
+def test_get_app_and_version(backend, backend_name, firmware):
+    client = BoilerplateCommand(backend)
+
+    app_name, version = client.get_app_and_version()
+
+    if backend_name == "speculos":
+        # for now, Speculos always returns app:1.33.7, so this test is not very meaningful on it
```

Comment:
Can be override by ENV variable :https://github.com/LedgerHQ/app-boilerplate/pull/18#discussion_r1026185012
But not sure it's worth it...

Reply:
Not worth it IMO